client/extensions/woocommerce/app/store-stats/index.js

```diff
@@ -36,8 +43,8 @@ class StoreStats extends Component {
 				<Navigation unit={ unit } type="orders" slug={ slug } />
 				<Chart
 					path={ path }
-					query={ ordersQuery }
-					selectedDate={ selectedDate }
+					query={ Object.assign( {}, ordersQuery, { date: unitQueryDate } ) }
```

Comment:
As I understand it, the DatePicker component below still needs the `ordersQuery` component with the date in a `YYYY-MM-DD` format. So instead of creating 2 different objects, I thought I'd update the `date` for the chart component.